nixos/modules/services/misc/airsonic.nix

```diff
@@ -0,0 +1,119 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.airsonic;
+in {
+  options = {
+
+    services.airsonic = {
+      enable = mkEnableOption "Airsonic";
```

Comment:
How about "Airsonic, the Free and Open Source media streaming server (fork of Subsonic and Libresonic)"?